docs/components/xr-mode-ui.md

```diff
@@ -19,25 +19,52 @@ to the [`<a-scene>` element][scene]. If we wish to simply toggle the UI, use CSS
-| enterVREnabled        | If the VR button is displayed when applicable                       | false         |
-| enterARButton         | Selector to a custom AR button. On click, the button will enter AR. | ''            |
-| enterAREnabled        | If the AR button is displayed when applicable                       | false         |
-| XRMode                | If the AR, VR button or both will be displayed.  One of 'ar', 'vr' or 'xr'.| vr            |
+| Property                | Description                                                         | Default Value | One of |
+|-------------------------|---------------------------------------------------------------------|---------------|--------|
+| `cardboardModeEnabled`  | Enables the now deprecated cardboard mode.                          | `false`       | -
+| `enabled`               | Whether or not to display UI related to entering VR.                | `true`        | -
+| `enterVRButton`         | Selector to a custom VR button. On click, the button will enter VR. | `''`          | -
+| `enterVREnabled`        | If the VR button is displayed when applicable.                      | `true`        | -
+| `enterARButton`         | Selector to a custom AR button. On click, the button will enter AR. | `''`          | -
+| `enterAREnabled`        | If the AR button is displayed when applicable.                      | `true`        | -
+| `XRMode`                | If the AR button, VR button, or both buttons will be displayed.     | `vr`          | `ar`, `vr`, `xr`
 
 ### Specifying a Custom Enter VR Button
 
+Display only a custom Enter VR button when VR is supported. A custom AR button will not be displayed.
+
+```html
+<a-scene
+  xr-mode-ui="enterVRButton: #myEnterVRButton;">
+  <!-- Style the button with images and/or custom CSS. -->
+  <a id="myEnterVRButton" href="#"></a>
+</a-scene>
+```
+
+### Specifying a Custom Enter AR Button
+
+Display only a custom Enter AR button when AR is supported. A custom VR button will not be displayed.
+
+```html
+<a-scene
+  xr-mode-ui="enterARButton: #myEnterARButton; XRMode: ar;">
+  <!-- Style the button with images and/or custom CSS. -->
+  <a id="myEnterARButton" href="#"></a>
+</a-scene>
+```
+
+### Specifying Custom Enter VR and Enter AR Buttons
+
+Display a custom Enter VR Button when VR is supported and a custom Enter AR button when AR is supported.
+
 ```html
 <a-scene
-  xr-mode-ui="enterVRButton: #myEnterVRButton; enterARButton: #myEnterARButton">
-  <!-- Style the button with images or whatever. -->
+  xr-mode-ui="enterVRButton: #myEnterVRButton; enterARButton: #myEnterARButton; XRMode: xr;">
+  <!-- Style the buttons with images and/or custom CSS. -->
   <a id="myEnterVRButton" href="#"></a>
   <a id="myEnterARButton" href="#"></a>
 </a-scene>
 ```
+> ( i ) This is similar to the behavior of the `vr-mode-ui` component that the `xr-mode-ui` component replaced in A-Frame 1.5.0.
```

Comment:
We use the deprecations section in release notes for this: https://github.com/aframevr/aframe/releases

Reply:
Try to avoid info in docs info that loses relevance or gets obsolete

Reply:
Thank you. Removed the note from this section.